AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerService.kt

```diff
@@ -294,6 +294,17 @@ interface TestRunnerService {
         /**
          * Test case log files produced by the test.
          */
-        val testCaseLogcats: Map<TestIdentifier, ResultFileResource>
+        val testCaseArtifacts: Map<TestIdentifier, List<TestResultLog>>
     }
+
+    data class TestResultLog(
```

Comment:
why do we call this Log if it is not necessarily a Log?

maybe TestArtifact ? TestResultArtifact? TestCaseArtifact?